plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/HttpUtil.java

```diff
@@ -72,20 +74,26 @@ public static int sendPostMultipart(String urlString, Map<String, String> parame
     return response.getStatusCode();
   }
 
-  public static int sendPost(String urlString, Map<String, String> parameters) throws IOException {
+  static int sendPost(String urlString, Map<String, String> parameters) throws IOException {
     String parametersString = getParametersString(parameters);
+    return sendPost(urlString, parametersString, "application/x-www-form-urlencoded");
+  }
+
+  static int sendPost(String urlString, String body, String mediaType)
+      throws MalformedURLException, IOException, ProtocolException {
```

Comment:
nit: the `throws` clause is not consistent with the other `sendPost(urlString, parameters)` method above.

Reply:
done